pkg/controller/utils/clientwrapper.go

```diff
@@ -119,11 +119,12 @@ func (cmt *ControllerMetricsTripper) RoundTrip(req *http.Request) (*http.Respons
 		metricKubeClientRequestSeconds.WithLabelValues(cmt.Controller.String(), req.Method, path, remoteStr, resp.Status).Observe(applyTime)
 		if applyTime >= 5.0 {
 			log.WithFields(log.Fields{
-				"controller": cmt.Controller.String(),
-				"method":     req.Method,
-				"path":       path,
-				"remote":     remoteStr,
-				"status":     resp.Status,
+				"controller":    cmt.Controller.String(),
+				"method":        req.Method,
+				"path":          path,
+				"remote":        remoteStr,
+				"status":        resp.Status,
+				"elapsedMillis": applyTime * 1000, // millis for consistency with how we log controller reconcile time
```

Comment:
Change `applyTime` to be a `time.Duration`, and use `applyTime.Milliseconds()` here. That way the user does not have to figure out that `applyTime` is in seconds.

Reply:
done